pkgs/by-name/gl/glog/package.nix

```diff
@@ -19,6 +20,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-+nwWP6VBmhgU7GCPSEGUzvUSCc48wXME181WpJ5ABP4=";
   };
 
+  postPatch = ''
```

Comment:
You could also guard this with `lib.optionalString finalAttrs.doCheck`, but that would obviously require a switch to the `finalAttrs` pattern as well.